jmx-metrics/src/main/groovy/io/opentelemetry/contrib/jmxmetrics/InstrumentHelper.groovy

```diff
@@ -118,7 +118,7 @@ class InstrumentHelper {
             def description = tuple.getAt(2)
             def unit = tuple.getAt(3)
 
-            if (instrumentIsObserver(instrument)) {
+            if (instrumentIsDoubleObserver(instrument) || instrumentIsLongObserver(instrument)) {
```

Comment:
Thanks for taking care of this. We have unit tests for the helpers that would need to be updated: https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-metrics/src/test/java/io/opentelemetry/contrib/jmxmetrics/InstrumenterHelperTest.java#L628

Reply:
Yup! Just saw those and pushed a change.